### DISCUSSION TITLE
Does not work

### DISCUSSION THREAD

**Discussion by Scraayp** [requarks/wiki#2844] [Help / Questions] [answered]
I want to disable options of share and view history of pages. How to do so?

from this menu btw
![image](https://user-images.githubusercontent.com/48729921/102716958-73a0f280-42df-11eb-9985-163019ff4dde.png)

Os = Linux
Wiki.js version = 2.5.170
Database engine = postgres

**Comment by NGPixel** [comment, marked as answer]
Not possible at the moment. Coming in v3.
